File: fabric-backend-core/src/main/java/io/storydoc/fabric/snapshot/app/descriptor/SnapshotDescriptorDTO.java
package io.storydoc.fabric.snapshot.app.descriptor;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class SnapshotDescriptorDTO {

    private String name;

    private String environmentKey;

    private List<SnapshotDescriptorItemDTO> componentDescriptors;

}

File: fabric-frontend/src/app/api/models/snapshot-descriptor-dto.ts
/* tslint:disable */
/* eslint-disable */
import { SnapshotDescriptorItemDto } from './snapshot-descriptor-item-dto';
export interface SnapshotDescriptorDto {
  componentDescriptors?: Array<SnapshotDescriptorItemDto>;
  environmentKey?: string;
  name?: string;
}
